build.cmd

```diff
@@ -1,6 +1,12 @@
 @echo off
 setlocal
 
+:: Call setup_vs_tools with the '1' flag to tell it to only check for the
+:: VS installation, and not launch the Dev Prompt. More details are in that
+:: script source file.
+
+call "%~dp0src\coreclr\setup_vs_tools.cmd" 1
```

Comment:
That doesn't seem right. We don't want to call into a coreclr script from the repo root. Can you extract that and move it into eng/?

Reply:
Also, isn't there an Arcade script that already does that?

Reply:
I did this in order to avoid code duplication, but now that you bring it up, it is probably better practice to keep root script functionality outside of any subsets of the repo. I can extract the checking functionality into a new script in `eng`. As far as I'm concerned, we do need to add this dependency check.

Reply:
yes please do so.

Reply:
and please check if Arcade already offers something useful to check dependencies. And if not, we might want to add it to Arcade.

Reply:
We have https://github.com/dotnet/runtime/issues/1251 opened on unifying this.

Arcade has https://github.com/dotnet/runtime/blob/master/eng/common/tools.ps1 that you probably had in mind. It is pretty far from what we need here.